### DISCUSSION TITLE
Does the Octopus Energy integration support more than one account?

### DISCUSSION THREAD

**Discussion by markcocker** [BottlecapDave/HomeAssistant-OctopusEnergy#656] [General]
I have the integration installed and working fine for one Octopus account.
I am trying to add sensors for a second Octopus account for a family member in the same HA instance.
I tried Settings > Devices and Services > Octopus Energy > Add Entry > however this starts a wizard to setup a target rate period.
Is it possible to add more than one Octopus account using this integration?

**Comment by BottlecapDave** [comment]
Hello. The integration currently only supports one account. Please upvote this [issue](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/223) so I can gauge interest and prioritise as it will require a bit of rework and testing to support.

**Comment by markcocker** [comment]
Thanks - I'll close this issue and follow the issue
